### PR TITLE
Fix/explore line

### LINKED ISSUE #29: Explore Page Spacing and Box Shadow
**Summary**

On the explore page, there is a line cutting the box shadow of the discover cards.

The space between the discover cards should be reduced.

---

### PR DESCRIPTION
**Pull request by skeshavaa**
**Summary**

Fixed explore line white line and spacing between cards.
---

**Closing issues**

Fixes #29

### CODE REVIEW

File: lib/screens/seeker-screens/explore.dart [hunk 1]
```diff
@@ -1,14 +1,23 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
 import 'package:main/screens/widgets/jobCard.dart';
 
 class Explore extends StatelessWidget {
 
-  final List job;
+  final String category;
+  final QuerySnapshot jobs;
```
Comment (richardxhong) [inline lib/screens/seeker-screens/explore.dart]: Shouldn't be final

File: lib/screens/seeker-screens/explore.dart [hunk 2]
```diff
@@ -18,10 +27,29 @@ class Explore extends StatelessWidget {
...
+      //         } else {
+      //           return Container(height: 0,);
+      //         }
+      //       },
+      //       itemCount: 2,
+      //     );
+      //   },
+      // )
+      
+      body: ListView(children: _actual)
```
Comment (richardxhong) [inline lib/screens/seeker-screens/explore.dart]: Remove unused code

File: lib/screens/seeker-screens/history.dart [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:main/screens/widgets/jobCard.dart';
+import 'package:provider/provider.dart';
+
+class History extends StatelessWidget {
+  
+  @override
+  Widget build(BuildContext context) {
+
+    var jobs = Provider.of<QuerySnapshot>(context);
```
Comment (richardxhong) [inline lib/screens/seeker-screens/history.dart]: Should be private

File: lib/screens/seeker-screens/historyDetails.dart [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+              padding: EdgeInsets.fromLTRB(MediaQuery.of(context).size.width * 0.1, 0, MediaQuery.of(context).size.width * 0.1, 0),
+            )
+          ],
+        ),
+        Container(
+          margin: new EdgeInsets.only(top: MediaQuery.of(context).padding.top),
+          child: new BackButton(color: Colors.white),
+        ),
+      ],
+    )));
```
Comment (richardxhong) [inline lib/screens/seeker-screens/historyDetails.dart]: need trailing commas, maybe configure a linter to automatically add them for you

File: pubspec.yaml [hunk 5]
```diff
@@ -20,6 +20,7 @@ dependencies:
   flutter:
     sdk: flutter
   flappy_search_bar: 1.4.1
+  intl: ^0.16.1
```
Comment (richardxhong) [inline pubspec.yaml:23]: What's intl?

**Review by richardxhong — CHANGES_REQUESTED** [review]

File: lib/screens/seeker-screens/explore.dart (hunk 1 above)
Comment (skeshavaa) [reply lib/screens/seeker-screens/explore.dart]: Resolved

File: lib/screens/seeker-screens/explore.dart (hunk 2 above)
Comment (skeshavaa) [reply lib/screens/seeker-screens/explore.dart]: Resolved

File: lib/screens/seeker-screens/history.dart (hunk 3 above)
Comment (skeshavaa) [reply lib/screens/seeker-screens/history.dart]: Resolved

File: lib/screens/seeker-screens/historyDetails.dart (hunk 4 above)
Comment (skeshavaa) [reply lib/screens/seeker-screens/historyDetails.dart]: Resolved

**Pull request closed without merging**